Xero.NetStandard.OAuth2/Model/Accounting/Invoice.cs

```diff
@@ -191,9 +191,9 @@ public enum StatusEnum
         public string InvoiceNumber { get; set; }
 
         /// <summary>
-        /// ACCREC only – additional reference number (max length &#x3D; 255)
+        /// ACCREC only – additional reference number
         /// </summary>
-        /// <value>ACCREC only – additional reference number (max length &#x3D; 255)</value>
+        /// <value>ACCREC only – additional reference number</value>
```

Comment:
Please mention this in the release note. 

### Accounting API
- Removed max length reference in the comments of InvoiceNumber in Invoice.cs